examples/hackweek_demo/run_demo.py

```diff
@@ -20,22 +20,14 @@
 # None to view all
 view_window = None
 
-# set up streamlit
-# creating a single-element container
-placeholder = st.empty()
 
 # %%
 # near real-time / live feed simulation
 
 ## Get transactions from data
 
 
-def get_ticker(data):
-    """Given transaction data, return a subset of the dataframe"""
-    # Return reverse of methods to put most recent transactions at the top
-    out = data[["blockNumber", "input_method"]].set_index("blockNumber").iloc[::-1]
-    return out
-
+# TODO should these all be seperate figures instead of one figure?
```

Comment:
with matplotlib I like to have 1 master `figure` and use gridspec to layout the subplots. But for now lets just leave this as a TODO.

Reply:
It's mostly how we want to organize stuff wrt streamlit. Streamlit also has container objects (and ways to put things in places), which require separate figures